TransShiftMex/Audapter.cpp

```diff
@@ -1356,6 +1361,41 @@ void *Audapter::setGetParam(bool bSet,
                     p.timeDomainPitchShiftAlgorithm,
                     p.sr, p.frameLen, p.timeDomainPitchShiftSchedule));
         }
+
+		if (bRemakeFilter) {
+			if (p.sr < 24000) {
```

Comment:
The filter coefficients below work only for a specific sampling rate, right? This is the case for the default sampling rate. We apply a low pass filter at the Nyquist frequency to avoid aliasing. So L1366 should be doing and `==` comparison instead.

Reply:
Let's say someone is sampling at a high frequency like 32kHz (96 kHz / 3 downfact). Their frequencies about the Nyquist frequency would go through the low pass filter either way, and neither filter is designed to work at 32 kHz. But wouldn't we still want them to use the 24 kHz filter, so they get data from the 16-24 kHz range?

This is an unlikely use case, so I don't have strong feelings either way.